supabase/migrations/20260816201747_sso_providers_block_direct_active_insert.sql

```
@@ -0,0 +1,76 @@
+-- Block direct PostgREST inserts of already-active / enforce_sso SSO providers.
+-- Private create path uses the user JWT client and only inserts pending_verification.
```

SQLFluff (4.3.0)

[error] Line is too long (86 > 80).

(LT05)